client/css/style.css

```diff
@@ -1495,6 +1495,15 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 }
 
 #connection-error {
+	font-size: 12px;
+	line-height: 36px;
+	font-weight: bold;
+	letter-spacing: 1px;
+	word-spacing: 3px;
+	text-transform: uppercase;
+	background: #e74c3c;
+	color: #fff;
+	text-align: center;
```

Comment:
Looks like a button, is not a button 😅
When testing a couple days ago, I ended up in a situation where the banner would show up, but the app would never reload. I had to close it and re-open it. Should this banner stay a button?

Reply:
Are you sure about not reloading? socket.io will try to reconnect for an infinite amount of time.

Reply:
I'm honestly not sure what happened precisely no. It acts super weirdly when running server and client from my laptop (not your PR's fault, it's the case on `master` as well) and that's all I tested so far. It could very well be macOS trying to be too smart.
I need to test with multiple devices and see how things go.

Reply:
It looks like our server close cleanup now causes the client to stop reconnecting (io server disconnect), I will look into it.

Reply:
It only looks like a button because it previously was a button. It actually doesn't really look like a button on it's own.